docs/source/model_design/risk_factors.rst

```diff
@@ -198,68 +198,115 @@ Confounding
+In order for a factor to be a confounder (C), it must meet the following criteria:
 
-**Confounding occurs when an estimate of the association between an exposure and a disease is mixed up with the effect of another exposure on the same disease, and the two exposures are associated.** Failure to account for confounding leads to incorrectly concluding that the effect is due to one, rather than the other variable. 
+  1. Must be a risk factor, or the surrogate of a risk factor, for the outcome (O) of interest in those who are unexposed to the exposure (E) being investigated.
+  2. Must be correlated, positively or negatively, with the exposure (E). If the study population is stratified into exposed and unexposed groups, this means that (C) has a differential distribution in the two groups.
+  3. In addition, (C) should not be an intermediate step on the causal pathway between the exposure (E) and outcome (O) being investigated (else the factor is considered an :ref:`intermediate <Intermediates>`). In other words, (C) should not be caused by the exposure (E).  
 
-In order for a factor to be a confounder, it must meet the following criteria:
-1. The factor (C ) must be associated with the exposure (E) being investigated,
-2. The factor (C) must be a risk factor for the outcome of interest in those who are not exposed to the explosure (E) being investigated,
-3. In addition, the factor (C) should not be on the causal pathway between the exposure (E) and disease (D) being investigated.
+.. todo::
+  insert link to intermediates
+
+.. image:: confounding_triangle.svg
+
+In **randomized experimental studies**, the process of randomising individuals (or groups) to different exposures generally ensures that the different groups are equally balanced with respect to all relevant factors that might influence the risk of the outcome. In such randomised studies, 'exposure' usually refers to a treatment or other intervention that is being compared to another, or to no intervention. Randomisation ensures that every treatment group has a similar risk of the outcome at the beginning of the study. Provided the study is conducted rigorously and is sufficiently large, if we see a difference in the incidence of the outcome between treatment groups at the end of the study, then we can conclude that this difference is caused by the treatment. For this reason, experimental studies provide the strongest evidence of a causal association between an exposure and disease.
```

Comment:
I think that these two sections (on randomized experimental and observational studies) can be cut, and you could go straight into your example. what do others think? @alibow ? @NathanielBlairStahn ?

Reply:
Actually---i see how this might be helpful when trying to find data, and assess the studies from which you're pulling effect sizes.

Reply:
I agree, but I think that this text is really useful to have elsewhere in our documentation. I'm not exactly sure where we should put it, but maybe we can save it and then discuss in an upcoming section where we would like to put general epi stuff like this? I think that it's a little distracting in our risk factors section as is!

Reply:
Ok! ill leave it in here, we can copy and paste as we see fit later!

Reply:
We should add a `.. todo::` to remind us to do this later.

Reply:
I think it would be good to have a study design page for our documentation and the important things to consider when pulling effect sizes from different study designs. I've made this into a todo note.